### DISCUSSION TITLE
Feature Request(s) : Columns + "Install Date"

### DISCUSSION THREAD

**Discussion by seeu3** [ebkr/r2modmanPlus#602] [Ideas]
Thanks for a great program!

REQUEST:
Please allow "Mod Name", "Author Name", "Disabled?", and "Install Date" to exist within columns, that can be clicked to toggle sorting by that column, ascending or descending.

And, even if Columns cannot be implemented (too much work, etc), please allow "Install Date" to be a piece of data saved by your mod manager. 

Why? 
(a) Because once you install dozens of mods for one game, and you have just installed a new mod, it would help save time finding what you just installed. 

(b) Also, sometimes mod authors state one name on a website, yet the mod installs as a different name. Sorting by install date would help this problem.

Cheers!

**Comment by ebkr** [comment]
Hi, thanks for the feedback!

Default behaviour is install date descending (oldest first)
As long as you don’t move the items around, they’ll be in order. You’ll find this under the “Custom” sort in the top right. You can make it newest first by switching “Standard” to “Reverse”.

In terms of columns, it doesn’t really work without a bunch of visual changes that would then make it feel very similar to most other mod managers. I’m personally not a huge fan of it and I think with the manager appearing fairly non-rigid by ditching columns helps to not overwhelm people (if that makes sense?).

> Also, sometimes mod authors state one name on a website, yet the mod installs as a different name. Sorting by install date would help this problem

Do you have an example of this? Everything should match the equivalent TS upload.
